Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/BackLighting.azsli

```diff
@@ -49,26 +52,44 @@ float3 GetBackLighting(Surface surface, LightingData lightingData, float3 lightI
+                thickness = max(transmissionDistance, surface.transmission.thickness);
                 float transmittance = pow( saturate( dot( lightingData.dirToCamera, -normalize( dirToLight + surface.normal * transmissionParams.z ) ) ), transmissionParams.y ) * transmissionParams.w;
                 float lamberAttenuation = exp(-thickness * transmissionParams.x) * saturate(1.0 - thickness);
                 result = transmittance * lamberAttenuation * lightIntensity;
             }
             break;
 
         case TransmissionMode::ThinObject:
-            // Thin object mode, using thin-film assumption proposed by Jimenez J. et al, 2010, "Real-Time Realistic Skin Translucency"
+            // Thin object mode, based on Jimenez J. et al, 2010, "Real-Time Realistic Skin Translucency"
             // http://www.iryoku.com/translucency/downloads/Real-Time-Realistic-Skin-Translucency.pdf
-
-            float litRatio = 1.0 - shadowRatio;
-            if (litRatio)
+            
             {
-                const float thickness = surface.transmission.thickness * transmissionParams.w;
+                // transmissionDistance < 0.0f means shadows are not enabled --> avoid unnecessary computation
+                if (transmissionDistance < 0.0f)
+                {
+                    break;
+                }
+
+                // Irradiance arround surface point. 
+                // Increase angle of influence (angle(N,L) -> angle(N,L) + acos(transmissionNdLBias)) to smooth transition regions
+                float3 E = surface.albedo * max(lightingData.transmissionNdLBias + dot(-surface.normal, dirToLight),0.0);
+
+                // Transmission distance modulated by hardcoded constant (could be exposed as a weight of scattering distance for transmission)
+                float s = transmissionDistance * 100.0;
+                
+                // Use scattering color to weight thin object transmission color
                 const float3 invScattering = rcp(transmissionParams.xyz);
-                const float falloff = ThinObjectFalloff(surface.normal, dirToLight);
-                result = TransmissionKernel(thickness, invScattering) * falloff * lightIntensity * litRatio;
+                
+                // Albedo at front (surface point) is used to approximate irradiance at the back of the object
+                // See observation 4 in [Jimenez J. et al, 2010] 
+                result = TransmissionKernel(s, invScattering) * lightIntensity * E * transmissionParams.w;
+                
+                // Alternative specific to skin translucency
+                // result = T(s) * lightIntensity * surface.albedo * E * transmissionParams.w;
+                
+                // Distance attenuation applied to hide artifacts due to low-res projected areas onto shadowmaps (might need some work in the future)
+                result *= 1.0 / pow(max(1.0, sqrt(attenuationDistance)), lightingData.distanceAttenuation + 1.0);
```

Comment:
nit: not sure that we need the sqrt and the power in this - both are expensive yet attenuation can be linear / cubic with good results.
Have you considered / tested?